library/src/main/java/com/datatorrent/lib/io/fs/FSInputModule.java

```diff
@@ -107,6 +110,9 @@ public void populateDAG(DAG dag, Configuration conf)
       dag.setAttribute(blockReader, Context.OperatorContext.PARTITIONER, new StatelessPartitioner<FSSliceReader>(readersCount));
       fileSplitter.setBlocksThreshold(readersCount);
     }
+    if (blocksThreshold != 0) {
```

Comment:
Do you need this check as blocksThreshold is always >=1 according to constraint?